src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -32,24 +32,24 @@
 abstract class DoctrineType extends AbstractType
 {
     /**
-     * @var ManagerRegistry
+     * @var DoctrineChoiceLoader[]
      */
-    protected $registry;
+    private static $choiceLoaders = array();
```

Comment:
IMO, this should not be static. We need separate caches for the ORM and ODM types